apps/system/lockscreen/js/charging.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
```

Comment:
Unfortunately files should be prefixed with 'lockscreen_' since we are still in the system app.